services/autorust/codegen/src/identifier.rs

```diff
@@ -148,10 +157,33 @@ mod tests {
     use super::*;
     use heck::SnakeCase;
 
+    #[test]
+    fn test_unicode() -> Result<(), Error> {
+        assert_eq!(unicode(',', false), "u2c");
+        Ok(())
+    }
+
+    #[test]
+    fn test_replace_first() -> Result<(), Error> {
+        assert_eq!(replace_first(".", false), "u2e");
+        assert_eq!(replace_first("/", false), "u2f");
```

Comment:
For the keyvault example, the enum value would be `U2f` for the slash. https://www.unicodepedia.com/unicode/basic-latin/2f/solidus/.